libs/fluid-elements/checkbox/src/lib/checkbox.spec.ts

```diff
@@ -218,6 +218,27 @@ describe('Fluid checkbox', () => {
     });
   });
 
+  describe('value', () => {
+    it('should set the default value to on', async () => {
```

Comment:
please add a test scenario where no value is present

Reply:
This is the test where no value is present. The default based on the MdnDocumentation is `on`. 
@rowa-audil maybe reword the tests `it` to `should have the correct default set`.